src/diffusers/models/attention.py

```diff
@@ -372,10 +372,12 @@ def forward(
             hidden_states = attn_output + hidden_states
 
         # 4. Feed-forward
-        if self.use_ada_layer_norm_continuous:
-            norm_hidden_states = self.norm3(hidden_states, added_cond_kwargs["pooled_text_emb"])
-        elif not self.use_ada_layer_norm_single:
-            norm_hidden_states = self.norm3(hidden_states)
+        # i2vgen doesn't have this norm 🤷‍♂️
+        if hasattr(self, "norm3") and self.norm3 is not None:
```

Comment:
Let's refactor this and all if-else statements that following to use `if self.norm_type in ["...."]:`

Reply:
Done in be518c87.